docs/server_guide.md

```diff
@@ -144,7 +144,7 @@ rss.server.flush.localfile.threadPool.size 20
 rss.server.flush.hadoop.threadPool.size 20
```

Comment:
This should be increased to > 30?

Reply:
You're right, do you have a suggested value? how about we set it to 50?

Reply:
Our production environment is set to 100.

Reply:
> You're right, do you have a suggested value? how about we set it to 50?

2 * 30 = 60?

Reply:
> > You're right, do you have a suggested value? how about we set it to 50?
> 
> 2 * 30 = 60?

Ok.